### DISCUSSION TITLE
[WASM] Image from url is not shown

### DISCUSSION THREAD

**Discussion by FrozDark** [unoplatform/uno#9538] [Q&A]
### Current behavior

Images from url is not shown. Using latest dev builds 4.5.0-dev.523. But it works on playground https://playground.platform.uno/#6b758242

![ujt3](https://user-images.githubusercontent.com/6670150/185399231-6e69124c-eccb-40e4-a4b5-1584c4d48edf.jpg)

### Expected behavior

Images should be shown as in the playground https://playground.platform.uno/#6b758242
![ujt3](https://user-images.githubusercontent.com/6670150/185399339-202f3b53-acf9-41d0-af5e-0da1a01f5a15.jpg)

### How to reproduce it (as minimally and precisely as possible)

The repro

[WasmUrlImageIssue.zip](https://github.com/unoplatform/uno/files/9373937/WasmUrlImageIssue.zip)

### Works on UWP/WinUI

Yes

### Environment

Uno.WinUI / Uno.WinUI.WebAssembly / Uno.WinUI.Skia

### Affected platforms

WebAssembly

### IDE

Visual Studio 2022

### IDE version

17.3.1

**Comment by jeromelaban** [comment]
Thanks for the report. Can you take a look at the browser debugger (the network tab) and see if the request finished properly? Also, can you take a look at the `img` node in the DOM and see if the image is the right one?

**Comment by FrozDark** [comment]
> Thanks for the report. Can you take a look at the browser debugger (the network tab) and see if the request finished properly? Also, can you take a look at the `img` node in the DOM and see if the image is the right one?

The image in the DOM is right
I looked at the request and it fails by the cors policy. But I wonder why it works in the playground? http://playground.platform.uno/#49ae000e

![cors](https://user-images.githubusercontent.com/6670150/185411515-90991373-3a75-4435-a97e-96d7223fea7d.jpg)

I've also seen an exception in the browser console from ImageBrush if you need
```
fail: Uno.UI.Dispatching.CoreDispatcher[0]
      Dispatcher unhandled exception
System.ApplicationException: [object Event]
   at Microsoft.UI.Xaml.Media.ImageBrush.<SetNaturalImageSize>d__46.MoveNext() in C:\a\1\s\src\Uno.UI\UI\Xaml\Media\ImageBrush.wasm.cs:line 223
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__128_0(Object state) in D:\a\Uno.DotnetRuntime.WebAssembly\Uno.DotnetRuntime.WebAssembly\runtime\src\libraries\System.Private.CoreLib\src\System\Threading\Tasks\Task.cs:line 1914
   at Uno.UI.Dispatching.CoreDispatcherSynchronizationContext.<>c__DisplayClass3_0.<Post>b__0() in C:\a\1\s\src\Uno.UI.Dispatching\Core\CoreDispatcherSynchronizationContext.cs:line 25
   at Uno.UI.Dispatching.CoreDispatcher.InvokeOperationSafe(UIAsyncOperation operation) in C:\a\1\s\src\Uno.UI.Dispatching\Core\CoreDispatcher.cs:line 337
```

**Reply by jeromelaban** [reply]
It could be because your app is not running with HTTPS ?

**Reply by FrozDark** [reply]
> It could be because your app is not running with HTTPS ?

Maybe. I don't have much knowledge in WebAssembly. That made me forget about Developer Tools in browsers. What I'm doing now is just testing how it would work in wasm and debugging just in VS. 
To run app with HTTPS will take me more time into this. Right now wasm is not in priority.

Btw I forced cors headers to images and they are loaded and shown now. Thanks for you time and remembering me about browser's DevTools =)
